core/chains/evm/gas/models.go

```diff
@@ -164,6 +171,26 @@ func (e WrappedEvmEstimator) GetFee(ctx context.Context, calldata []byte, feeLim
 	return
 }
 
+func (e WrappedEvmEstimator) GetCost(ctx context.Context, amount assets.Eth, calldata []byte, feeLimit uint32, maxFeePrice *assets.Wei, opts ...feetypes.Opt) (*big.Int, error) {
+	fees, gasLimit, err := e.GetFee(ctx, calldata, feeLimit, maxFeePrice, opts...)
+	if err != nil {
+		return nil, err
+	}
+
+	var gasPrice *assets.Wei
+	if e.EIP1559Enabled {
+		gasPrice = fees.DynamicFeeCap.Add(fees.DynamicTipCap)
```

Comment:
What are you trying to evaluate here? This calculation seems nonsensical. `FeeCap` includes the tip cost.

Reply:
ah - that is my misinterpretation. i thought FeeCap == BaseFee